File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/InjuryProneTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class InjuryProneTag extends Tag {
    public InjuryProneTag(int count) {
        super(count, Constants.INJURY_PRONE_TAG, Constants.INJURY_PRONE_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/InjuryProneTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class InjuryProneTag(count: Int) : Tag(count, Constants.INJURY_PRONE_TAG, Constants.INJURY_PRONE_TAG_REMOTE, HashSet(listOf(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))